entrypoint.sh

```diff
@@ -5,4 +5,4 @@ if [ $# -ne 1 ]; then
 	exit 1
 fi
 
-dotnet run -p ./src/Render --input $1 --output /output
+dotnet run -p /render/src/Render/D2L.Dev.Docs.Render.csproj --input $1 --output output
```

Comment:
I haven't found a solution for this yet; the output directory is created in `/github/workspace`, this means that the repository can't have an output directory or this will break.
Attempts to write to `/output` and `/github/output` caused s3 to fail with `directory does not exist`.